projects/mysadna/src/app/components/contacts/contacts.component.scss

```diff
@@ -14,9 +14,9 @@
     padding-left: $margin;
     padding-top: $margin;
     text-align: start;
-    @media(max-width: 579px) {
-      text-align: center;  
-    }
+    // @media(max-width: 579px) {
+    //   text-align: center;  
+    // }
```

Comment:
Commented code?

Reply:
In case you decide to use it in the end.
Or should I simply remove it?

Reply:
When you push commit to PR, it's save here forever. So if we'll decide to use the code, we can take it from here.
But commented code in actual app make the app mess

Reply:
So you, could you remove it?